### PR TITLE
Update __init__.py

### PR DESCRIPTION
**Pull request by simpat1zq**
There are 3 lines changed. This is what I had to do in order to be able to change both color and brightness. In addition to that, I had to set the 'mode' in HA to RGB. In MagicHome, my device shows up as version 1.

### CODE REVIEW

File: flux_led/__init__.py [hunk 1]
```diff
@@ -661,7 +661,8 @@ def setRgb(self, r,g,b, persist=True, brightness=None):
         msg.append(int(g))
         msg.append(int(b))
         msg.append(0x00)
-        msg.append(0xf0)
+        msg.append(0x00)
+        msg.append(0x00)
```
Comment (tchellomello) [inline flux_led/__init__.py]: @simpat1zq thanks for your contribution. Is there any specific reason to have 3 sequential `msg.append(0x00)` entries?

Comment (simpat1zq) [inline flux_led/__init__.py]: Keep in mind that I'm not really a developer. @Danielhiversen asked me to do a pull request with the changes I had made in my file to get my device working. I don't know why there are 3 0x00's in there, but I actually removed it when I realized I had one too many(while I was submitting the first pull request), and my setup didn't work without. So this is what I have in my environment to get it working. I think that Daniel just wants to take a look at these changes to determine what to do with it. 

So long story short, I don't know why it needs 3 of them, but that's the only way it works with my device.

Comment (tchellomello) [inline flux_led/__init__.py]: @Danielhiversen  Thanks. yes I got it. sounds good.
